tasks/db_deploy/db_deploy.py

```diff
@@ -46,17 +46,21 @@ def handler(
     # Get the configuration
     config = get_configuration()
 
-    return task(config)
+    #get partition bucket list
+    partition_bucket = event["buckets"]
```

Comment:
This will create a partition for each bucket, not just the glacier buckets. At least that's what's suggested by the syntax.
If limiting to glacier buckets causes problems, the extra partitions won't break anything.

Reply:
Changed the input and added some specifics so now is `event["orcaBuckets"]` which in terraform we already have a list of orca only buckets we use for other lambdas/policies.